documentation/guide-openapi.asciidoc

```diff
@@ -0,0 +1,94 @@
+:toc: macro
+toc::[]
+
+= OpenAPI
+
+The link:https://spec.openapis.org/oas/latest.html[OpenAPI Specification (OAS)] defines a standard for describing RESTful web services in a machine- and human-readable format. OpenAPI allows REST APIs to be defined in a uniform manner.
+Technically, an OpenAPI document is written in YAML or JSON format. The specification defines the structure of a REST API by describing attributes such as path information, response codes, and return types. Some examples can be found link:https://github.com/OAI/OpenAPI-Specification/tree/main/examples/v3.0[here].
+
+OpenAPI is often used in combination with link:https://swagger.io/[Swagger]. Swagger is a set of tools build around OpenAPI, that help developers to design and document their REST APIs.
+The most common tool is the link:https://swagger.io/tools/swagger-ui/[Swagger UI], which uses the OpenAPI specification to create a graphical interface of the REST API that you can also interact with. Check out the link:https://editor.swagger.io/[Swagger online editor] to get a feeling of this.
+
+There are several extensions you can use in your project to automatically generate the OpenAPI specifications and the Swagger UI from your REST API (code first approach).
+
+== Springdoc-OpenAPI
+
+The link:https://springdoc.org/[`springdoc-openapi-ui`] extension can be used to add OpenAPI and Swagger UI support to Spring application. Just add the following extension to you pom.xml file:
+
+[source,xml]
+--------
+<dependency>
+  <groupId>org.springdoc</groupId>
+  <artifactId>springdoc-openapi-ui</artifactId>
+  <version>1.5.12</version>
+</dependency>
+--------
+
+The extension analysis the REST API classes at runtime and then generates the documentation from them. The documentation can be extended by using link:https://github.com/swagger-api/swagger-core/wiki/Annotations-1.5.X[Swagger API annotations].
```

Comment:
the extension analysis(-> analyzes) ...

Reply:
I think both spellings are correct, don't you?